src/App.module.css

```diff
@@ -0,0 +1,47 @@
+.panel_bg {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  border-bottom: solid 0.125rem #e8e8e8;
+}
+
+.container {
+  width: 87.5rem;
+  height: 68.563rem;
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: rgba(100, 100, 111, 0.2) 0 7px 29px 0;
+}
+
+h1 {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+a {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
+}
```

Comment:
- [ ] Try to get closer with the zeplin design. kindly you are missing some styling features as you can see on the original image.

---------------------------------------------Yours---------------------------------------------

![cppp](https://user-images.githubusercontent.com/55635977/132950428-19b1513d-7a49-47ef-a07b-ab147cdae466.png)
---------------------------------------------Original---------------------------------------------

![cppp1](https://user-images.githubusercontent.com/55635977/132950430-880c9fad-b86a-4430-8ded-bece2c53e519.png)